WooCommerce/WooCommerceTests/POS/ViewModels/ItemListViewModelTests.swift

```diff
@@ -407,7 +407,66 @@ final class ItemListViewModelTests: XCTestCase {
         await sut.loadInitialItems()
 
         // Then
-        XCTAssertEqual(receivedStates, [.loading, .loaded(items)])
+        XCTAssertEqual(receivedStates, [.initialLoading, .loaded(items)])
+    }
+
+    func test_sut_when_there_are_no_items_then_statePublisher_emits_expected_state() async {
+        let itemProvider = MockPOSItemProvider()
+        itemProvider.shouldReturnZeroItems = true
+        let sut = ItemListViewModel(itemProvider: itemProvider)
+
+        let expectation = XCTestExpectation(description: "Publisher should emit state changes")
```

Comment:
❓ My understanding is that an XCTest expectation has to be waited for by `wait(for:)` to take effect, which doesn't seem needed in these two new test cases since the load functions are already `async` 🤔 I tried removing the expectation in this and the next test case and tests seemed to pass locally. Do you plan to wait for the expectation?

Reply:
That makes a lot of sense 😅 we do not need to wait here and this can be removed: https://github.com/woocommerce/woocommerce-ios/pull/14226/commits/c07640129655f7fc364bc82e4ae1dd57f37a95a7